source/common/upstream/health_checker_impl.h

```diff
@@ -206,11 +206,11 @@ class HttpHealthCheckerImpl : public HealthCheckerImplBase {
     Http::ResponseHeaderMapPtr response_headers_;
     Buffer::InstancePtr response_body_;
     const std::string& hostname_;
-    const Http::Protocol protocol_;
     Network::ConnectionInfoProviderSharedPtr local_connection_info_provider_;
-    bool expect_reset_{};
-    bool reuse_connection_ = false;
-    bool request_in_flight_ = false;
+    const Http::Protocol protocol_;
```

Comment:
Can the type of [Protocol](https://github.com/envoyproxy/envoy/blob/1ac0a5948ca317fd418777dab2ffe27935c0a276/envoy/http/protocol.h#L12) be redefined?

Reply:
Talked offline, about setting it to uint8. Done